Assets/scripts/Player/GameApplication.cs

```diff
@@ -0,0 +1,141 @@
+﻿using Cameras;
+using Enemy;
+using Interfaces.ServiecesInterface;
+using Replay_Service;
+using ScriptableObjects;
+using StateMachines;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+namespace Player
+{
+    public class GameApplication : Singleton<GameApplication>
```

Comment:
why does this class have spawn/ respawn logic?

Reply:
Actually this is my player service I named it wrong